client/gutenberg/editor/calypsoify-iframe.jsx

```diff
@@ -237,6 +267,14 @@ class CalypsoifyIframe extends Component {
 						source=""
 						visible={ isMediaModalVisible }
 					/>
+					<EditorMediaModal
```

Comment:
Done this way, won't both components always be rendered, even if classic blocks are never used? Can we avoid that?

Reply:
Oh, looking deeper, I think I'm wrong 🙃 . Correct me if I'm wrong about being wrong.